packages/js-client/src/core.ts

```diff
@@ -0,0 +1,13 @@
+import { ClientCore } from "@aragon/sdk-client-common";
+import { SimpleStorageContext } from "./context";
+
+export class SimpleStorageClientCore extends ClientCore {
```

Comment:
What's the goal of this class? Don't we have the client class itself and the context class?

Is there a way to have state/settings + logic in two components instead of 3?

Reply:
Mmmm I'm afraid we can't, we have the context and the client, but then we have the core that is the one that has all the web3, graphql and ipfs modules and starts the clients, so all the modules must implement this, the easier solution i found is extending this class and exporting here the new parameters and the implementing this class in the methods, encodign, decoding and estimation modules so we have access to the parameters

Reply:
Why don't we export the ClientCore class so that plugins don't need to reinvent the wheel each time?
It's true that some might not need subgraph, but then they are free to just build their own client